pyanaconda/anaconda_log.py

```diff
@@ -130,6 +133,25 @@ class AnacondaFileHandler(_AnacondaLogFixer, logging.FileHandler):
 class AnacondaStreamHandler(_AnacondaLogFixer, logging.StreamHandler):
     pass
 
+class AanacondaPrefixFilter(logging.Filter):
+    """Add a log_prefix field, which is based on the name property,
+    but without the "anaconda." prefix.
+
+    Also if name is equal to "anaconda", generally meaning some sort of
+    general (or miss-directed) log message, set the log_prefix to "misc".
+    """
+
+    def filter(self, record):
+        record.log_prefix = ""
+        if record.name:
+            # messages going to the generic "anaconda" logger get the log prefix "misc"
+            if record.name == "anaconda":
+                record.log_prefix = "misc"
+            else:
+                # drop "anaconda." from the log prefix
+                record.log_prefix = re.sub(REMOVE_ANACONDA_PREFIX_RE, "", record.name)
```

Comment:
That's a good idea - and even though using a constant like this is slightly terrifying, this part of the code is indeed potentially quite performance sensitive & we are not likely to change the number any time soon. :)